app/workers/event_worker.rb

```diff
@@ -1,15 +1,143 @@
 # frozen_string_literal: true
 
+require 'net/http'
+require_relative '../lib/exceptions/large_status_code'
+
+SCHEME = 'http://'
```

Comment:
Why aren't these in the class? Is there a sidekiq reason?

Doing this pollutes the global namespace. HTTP_ERRORS and all the methods would be accessible anywhere in the app. These seem like private instance methods?

Reply:
I can put those in the class, no problem.